content/faq/bookmarks/how-to-import/index.ru.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Как импортировать метки?
+description: ""
+updated: "2024-06-12"
+
+taxonomies:
+  faq: ["Метки и Треки"]
+
+extra:
+  order: 20
+---
+
+Вы можете импортировать метки отправленные из Organic Maps или сторонних приложений, если они позволяют экспортировать метки в форматах KML, KMZ, GPX:
+
+1. На Android, вы можете импортировать метки из некоторых других приложений, включая Maps.Me. Откройте метки (кнопка со звёздочкой), нажмите "Импорт меток", и выберите диск или папку, где будут KML, KMZ, GPX файлы.
```

Comment:
```suggestion
1. Вы можете импортировать метки из некоторых других приложений, включая Maps.Me. Откройте метки (кнопка со звёздочкой), нажмите "Импорт меток и треков", и выберите диск или папку, где будут KML, KMZ, KMB или GPX файлы.
```

Reply:
1. Now iOS also has import button.
2. Translations for the button text can be taken from strings.txt

Reply:
Translated button "Import Bookmarks and Tracks" according to strings.txt
But Android is still mentioned in Espanol, German and other languages. We need to ask for translation.

Reply:
Thanks! I've fixed other langs.